File: PYTHON PROGRAMMING/Test33.py
# 33 PRINT THOSE NUMBER FROM RANGE1 TO RANGE2 WHICH ARE DIVISIBLE BY THE GIVEN TARGET
def func(limit1, limit2, target):
    for i in range(limit1, limit2 + 1):
        if(i % target == 0):
            print(i, end=" ")
    print()
func(10, 20, 3)
func(30, 50, 4)
func(60, 80, 7)

File: TYPESCRIPT PROGRAMMING/Test33.ts
// 33. PRINT THOSE NUMBER FROM RANGE1 TO RANGE2 WHICH ARE DIVISIBLE BY THE GIVEN TARGET
const func33 = (range1: number, range2: number, target: number):void => {
    for(let i = range1; i <= range2; i++) {
        if(i % target === 0) {
            process.stdout.write(`${i} `);
        }
    }
    console.log();
}
func33(20, 40, 4);
func33(50, 80, 12);